java/src/jmri/jmrit/display/switchboardEditor/BeanSwitch.java

```diff
@@ -110,7 +110,7 @@ public BeanSwitch(int index, @CheckForNull NamedBean bean, @Nonnull String switc
             allControlling = editor.allControlling();
             panelEditable = editor.isEditable();
             showToolTip = editor.showToolTip();
-            showUserName = editor.nameDisplay();
+            showUserName = editor.nameDisplay(); // 0,1,2
```

Comment:
What's the purpose of the comment `1,2,3`? Or is it a left over from before `showUserName` became an enum?

Reply:
Thanks.  I've removed that comment. Not sure why it was there.